Code/Mantid/Framework/DataHandling/src/LoadMcStas.cpp

```diff
@@ -543,24 +543,24 @@ void LoadMcStas::readHistogramData(
  */
 int LoadMcStas::confidence(Kernel::NexusDescriptor &descriptor) const {
   using namespace ::NeXus;
```

Comment:
A comment to test a return comment to Karl's comment.

(Found this functionality did not work in one version of internet explorer!)